client/containers/MovieSearchContainer.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect, useRef } from "react";
+import MovieTile from "../components/MovieTile";
+import axios from "axios";
+
+function MovieContainer(props) {
```

Comment:
change the name of this component to match name of file

Reply:
fair, I have updated it.